src/refactor_nrepl/util.clj

```diff
@@ -72,3 +72,14 @@
 (defn maybe-log-exception [^Throwable e]
   (when (System/getProperty "refactor-nrepl.internal.log-exceptions")
     (.printStackTrace e)))
+
+(defn wrap-ignore-errors [f ignore-errors?]
+  (if-not ignore-errors?
+    f
+    (fn [x]
+      (try
+        (f x)
+        (catch Exception e
+          (maybe-log-exception e)
+          ;; return false, because `wrap-ignore-errors` is oriented for predicate usage:
```

Comment:
the trailing `:` in this comment makes it seem like something is missing

Reply:
> the trailing : in this comment makes it seem like something is missing

I add the colon to make it clear that the comment refers to the next LOC. Else there can be an ambiguity (does it refer to the previous one, so the defn in general, etc?)

I guess it failed at that 😇